js/web/lib/backend-wasm-with-training.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {InferenceSession, TrainingSessionHandler} from 'onnxruntime-common';
+
+import {OnnxruntimeWebAssemblyBackend} from './backend-wasm';
+
+class OnnxruntimeTrainingWebAssemblyBackend extends OnnxruntimeWebAssemblyBackend {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
you can name the parameters ends with "_" ( or starts with, not remember which one ) to indicate an unused var instead of suppressing the linter